src/plugins/orangehrmCorporateDirectoryPlugin/lib/wrapper/CorporateDirectoryWebServiceHelper.php

```diff
@@ -45,7 +45,8 @@ public function setEmployeeService(EmployeeService $employeeService) {
      * @param type $includeTerminated
      * @return Array
      */
-    public function getCorporateDirectoryEmployeeDetailsAsArray($includeTerminated = false) {
+    public function getCorporateDirectoryEmployeeDetailsAsArray($includeTerminated = false): array
```

Comment:
not sure why this changed. need to delete these 4.x legacy classes

Reply:
Done